packages/sdk-ark/source/signed-transaction.dto.ts

```diff
@@ -101,6 +101,10 @@ export class SignedTransactionData
 	}
 
 	public override usesMultiSignature(): boolean {
+		if (!this.signedData) {
```

Comment:
This isn't a fix, just muting the problem. You need to figure out why `this.signedData` ends up being anything but the object it should be.

Reply:
There are a lot of side-effects in wallet's transaction state that will require more time to find out what it does and when. Any ideas?